kong/conf_loader.lua

```diff
@@ -473,7 +473,7 @@ local function parse_nginx_directives(dyn_key_prefix, conf)
           v = string.format("%q", v)
         end
 
-        directives[directive] = v
+        table.insert(directives, { name = directive, value = v })
```

Comment:
We could have preserved this behavior to keep the configuration output of the `/` Admin API endpoint in a key/value fashion, and make the array ordering an implementation detail for now, but well.

Reply:
Let's ignore this for now.